.github/workflows/ci.yml

```diff
@@ -22,31 +22,51 @@ jobs:
         run: |
           MODULES=$(find -name pom.xml | sed -e 's|pom.xml| |' | sed -e 's|./| |' | grep -v " quarkus/" | grep -v resources)
           CHANGED=""
+          LONG_RUNNING_TESTS_CHANGED=""
           MODULES_ARG=""
+          MODULES_LONG_RUNS_ARG=""
 
           # If changed file have some special character, its path is surrounded with quotes which causing the if statement fail
           CHANGED_FILE=$(echo ${{ steps.files.outputs.all_changed_and_modified_files }} | sed 's/\"/\\"/')
 
           for module in $MODULES
           do
             if [[ $CHANGED_FILE =~ ("$module") ]] ; then
-                CHANGED=$(echo $CHANGED" "$module)
+                # Check if changed file contains @Tag("long-running")
+                LONG_RUNNING_TAG_FOUND=false
+                for file in $(find "$module" -type f -name "*.java"); do
+                  if grep -q '@Tag("long-running")' "$file"; then
+                    LONG_RUNNING_TAG_FOUND=true
+                    break
+                  fi
+                done
+          
+                if $LONG_RUNNING_TAG_FOUND; then
```

Comment:
Looking at this and I think you work with outdated version of action file. We merged  https://github.com/quarkus-qe/quarkus-test-suite/pull/2201 last week so you probably need to update it.

Reply:
Thank you, I will rebase PR